adoc/admin-disaster-recovery-velero.adoc

```diff
@@ -0,0 +1,707 @@
+
+. SUSE Helm Chart
++
+Add SUSE helm chart repository URL
++
+[source,bash]
+----
+helm repo add suse https://kubernetes-charts.suse.com
+----
+
+. Object Storage
++
+Velero uses object storage to store backups and associated artifacts. It can optionally snapshot the persistent volume and stores in the object storage if the persistent volumes does not supports volume snapshot.
+
+.. Public Cloud Storage Providers
++
+|=======
+| Provider | Object Storage | Plugin Provider Repo
+| Amazon Web Services (AWS) | AWS S3 | https://github.com/vmware-tanzu/velero-plugin-for-aws[Velero plugin for AWS]
+| Google Cloud Platform (GCP) | Google Cloud Storage | https://github.com/vmware-tanzu/velero-plugin-for-gcp[Velero plugin for GCP]
+| Microsoft Azure | Azure Blob Storage | https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure[Velero plugin for Microsoft Azure]
+|=======
+
+.. S3-compatible Storage Providers
+
+... SES6 Ceph Object Gateway (radosgw)
++
+SUSE supports the SES6 Ceph Object Gateway (radosgw) as the S3-compatible storage provider. Refer to https://documentation.suse.com/ses/6/html/ses-all/cha-ceph-additional-software-installation.html[SES6 Object Gateway Manual Installation] on how to install it.
++
+... Minio
++
+For the evaluation install, there are opensource S3-compatible storage provider https://min.io/[minio]. Prepare an external host and install Minio on the host.
++
+[source,bash]
+----
+# Download Minio server
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
+chmod +x minio
+
+# Expose Minio access_key and secret_key
+export MINIO_ACCESS_KEY=<access_key>
+export MINIO_SECRET_KEY=<secret_key>
+
+# Start Minio server
+mkdir -p bucket
+./minio server bucket &
+
+# Download Minio client
+wget https://dl.min.io/client/mc/release/linux-amd64/mc
+chmod +x mc
+
+# Setup Minio server
+./mc config host add velero http://localhost:9000 $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+
+# Create bucket on Minio server
+./mc mb -p velero/velero
+----
+
++
+For the rests of the S3-compatible storage providers, refer to https://velero.io/docs/v1.2.0/supported-providers/[Velero Supported Providers].
+
+. Volume Snapshotter
++
+[NOTE]
+The volume snapshotter snapshots the persistent volume if the volume supports the snapshot API to do volume snapshot. If the volume does not supports snapshot API or does not have supported providers, Velero leverages _restic_ to do persistent volume backup and restore.
+
+[IMPORTANT]
+Please be well-known the persistent volume in the supported volume snapshotter provides. Then adminstrators have to determine whether to deploy the _restic daemonset_ or not.
+
+.. Public Cloud Storage Providers
++
+|=======
+| Provider | Volume Snapshotter | Plugin Provider Repo
+| Amazon Web Services (AWS) | AWS EBS | https://github.com/vmware-tanzu/velero-plugin-for-aws[Velero plugin for AWS]
+|=======
+
+.. Community Supported Providers
```

Comment:
I would suggest no need to list non-supported items and instead just have a link to external documentation.